metadata.rb

```diff
@@ -6,14 +6,12 @@
 long_description 'Installs/Configures stackstorm'
 version '0.4.0'
 
-supports 'debian'
-supports 'ubuntu'
-supports 'redhat'
-supports 'centos'
+supports 'ubuntu', '= 14.04'
+supports 'redhat', '~> 7.1'
+supports 'centos', '~> 7.1'
```

Comment:
It comes from the knowledge that it definitely works with `7.1`+.
I haven't tested with `7.0` yet. In fact, this might be a good call, let's check if that cookbook really works in `7.0` and change it later.